src/Kunstmaan/TranslatorBundle/Repository/TranslationRepository.php

```diff
@@ -62,9 +62,10 @@ public function getLastChangedTranslationDate()
         $flagNew = \Kunstmaan\TranslatorBundle\Entity\Translation::FLAG_NEW;
         $flagUpdated = \Kunstmaan\TranslatorBundle\Entity\Translation::FLAG_UPDATED;
 
+        //use lowercase variable names for postgress. i.e. : newest_date instead of newestDate
```

Comment:
Can you remove this comment. It makes sense in this PR and the context of adding postgres support, but this info is not needed after it's merged

Reply:
done